commit 538e67c660b2e1fb6f5cad8e6592aa084f812bb4
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Feb 9 14:36:44 2017 +0000

    sound: Demote deferral errors to INFO level
    
    At present there is no mechanism to specify driver load order,
    which can lead to deferrals and repeated retries until successful.
    Since this situation is expected, reduce the dmesg level to
    INFO and mention that the operation will be retried.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/sound/soc/soc-core.c b/sound/soc/soc-core.c
index 13c875e2392a..23d56057e49b 100644
--- a/sound/soc/soc-core.c
+++ b/sound/soc/soc-core.c
@@ -1072,7 +1072,7 @@ static int soc_bind_dai_link(struct snd_soc_card *card,
 	cpu_dai_component.dai_name = dai_link->cpu_dai_name;
 	rtd->cpu_dai = snd_soc_find_dai(&cpu_dai_component);
 	if (!rtd->cpu_dai) {
-		dev_err(card->dev, "ASoC: CPU DAI %s not registered\n",
+		dev_info(card->dev, "ASoC: CPU DAI %s not registered - will retry\n",
 			dai_link->cpu_dai_name);
 		goto _err_defer;
 	}
@@ -1084,7 +1084,7 @@ static int soc_bind_dai_link(struct snd_soc_card *card,
 	for (i = 0; i < rtd->num_codecs; i++) {
 		codec_dais[i] = snd_soc_find_dai(&codecs[i]);
 		if (!codec_dais[i]) {
-			dev_err(card->dev, "ASoC: CODEC DAI %s not registered\n",
+			dev_info(card->dev, "ASoC: CODEC DAI %s not registered - will retry\n",
 				codecs[i].dai_name);
 			goto _err_defer;
 		}
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1031_538e67c660b2e1fb6f5cad8e6592aa084f812bb4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1031_538e67c660b2e1fb6f5cad8e6592aa084f812bb4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
